Packages/vcs/Lib/VTKPlots.py

```diff
@@ -87,7 +90,7 @@ def endEvent(self,obj,event):
     if self.renWin is not None:
       if self.reRender:
         self.reRender = False
-        self._lastSize = None
+        #self._lastSize = None
```

Comment:
_lastSize was needed to work around some resizing bugs in the UI on mac. I have a feeling we'll see a regression if we take this out.

IIRC, @doutriaux1 put this in -- can you let us know if removing this would break that workaround?